### ISSUE TITLE
Typo in Select example

### ISSUE THREAD

**Issue by antongavrilenkov** [material-components/material-components-web#1839]
https://material.io/components/web/catalog/input-controls/select-menus/

There is a typo in the latest code example:

```js
const selectManager = document.querySelector('.select-manager');
const selects = {
  custom: MDCSelect.attachTo(selectManager.querySelector('.mdc-select[role="listbox"]')),
  native: MDCSelect.attachTo(selectManager.querySelector('select.mdc-select'))
};
const changeHandler = ({type}) =>  {
  let changedSelect, selectToUpdate, value;
  if (type === 'MDCSelect:change') {
    changedSelect = selects.custom;
    selectToUpdate = selects.native;
    value = changedSelect.selectedOptions[0].id;
  } else {
    changeSelect = selects.native; // <<<<<<<< TYPO IS HERE <<<<<<<<
    changedSelect = selects.native; // <<<<<<<< CHANGE TO THIS (add a 'd' to 'change')
    selectToUpdate = selects.custom;
    value = changedSelect.selectedOptions[0].value;
  }
  selectToUpdate.selectedIndex = changedSelect.selectedIndex;
  console.info('Selected value', value);
};
selects.custom.listen('MDCSelect:change', changeHandler);
selects.native.addEventListener('change', changeHandler);
```

**Comment by acdvorak** [comment]
Line 494 of [README.md](https://github.com/material-components/material-components-web/blob/master/packages/mdc-select/README.md)

**Comment by simono** [comment]
I fixed this in #1880.
